W-13630523 1.5 coverage status gr (#381)
mulesoft/docs-gateway: gateway/1.5/modules/ROOT/pages/policies-automated-applying.adoc

// +
// If an application is deployed to an environment which has an automated policy that doesn't cover the deployed runtime, for example with an automated policy affecting a runtime range from 4.1.2 to 4.1.4, and a user deploys to a 4.1.5 runtime, a notification will be sent to the organization, and environment administrators, to let them know of this activity.

=== Viewing Coverage Status

To view what API instances are covered by an automated policy:

. Navigate to *Anypoint Platform* > *API Manager*.
. In API Administration, click *Automated Policies*.

> . Click more options button (image:more-options-menu.png[1%,1%]) of the policies whose coverage you want to view.

Reviewer: ```suggestion
. Click the more options button (image:more-options-menu.png[1%,1%]) for the policy whose coverage you want to view.
```
I'm thinking they should be consistent in terms of being singular.

// +
// If an application is deployed to an environment which has an automated policy that doesn't cover the deployed runtime, for example with an automated policy affecting a runtime range from 4.1.2 to 4.1.4, and a user deploys to a 4.1.5 runtime, a notification will be sent to the organization, and environment administrators, to let them know of this activity.

=== Viewing Coverage Status

> View the *Coverage status* of an API instances to find if an automated policy covers your API instance or why it doesn't. Typically, instances are not covered due to their runtime type or runtime version.

Reviewer 2: ```suggestion
View the *Coverage status* of an API instance to find out if an automated policy covers your API instance or why it doesn't. Typically, instances are not covered due to their runtime type or runtime version.
```

To view what API instances are covered by an automated policy:

. Navigate to *Anypoint Platform* > *API Manager*.

> . In API Administration, click *Automated Policies*.

Reviewer 2: ```suggestion
. In *API Administration*, click *Automated Policies*.
```
